### DISCUSSION TITLE
Adding icons to buttons

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1137] [Q&A] [answered]
How can I add an icon to the button text?

![image](https://user-images.githubusercontent.com/6786299/116461653-235a1400-a82e-11eb-923a-11bcc91db363.png)

**Comment by meetmandeep** [comment, marked as answer]
I'm not sure if this is currently possible without using the Custom Code block. You may want to add this as a feature request.

**Reply by scottw12** [reply]
You know my view point on Custom Code Block I have NEVER gotten one to work right lol

**Reply by scottw12** [reply]
But ya a feature request as this is very common. Also icons in menus.

**Reply by meetmandeep** [reply]
Yes. I know you have another discussion open on Custom Code block. I'll verify that with nightly build of RC 2 and respond. I know many issues were fixed regarding Custom Code Block. 

I would add icons in menu as a separate feature request as well.
